app-frontend/src/app/components/lab/templateItem/templateItem.html

```diff
@@ -22,4 +22,14 @@ <h4 class="panel-title">{{$ctrl.templateData.title}}</h4>
       <strong>Created by: {{$ctrl.templateOwner}}</strong>
     </div>
   </div>
+  <div uib-dropdown class="dropdown template-delete-dropdown" ng-if="$ctrl.showDelete">
+    <button class="btn btn-tiny btn-ghost" uib-dropdown-toggle>
```

Comment:
I think we should remove the `btn-ghost` class. On the high-contrast theme especially, the buttons are a bit too subtle.

![image](https://user-images.githubusercontent.com/2442245/48414304-540a6980-e718-11e8-9059-9e7d37726605.png)

Reply:
will do. good call!